internal/wait/machineconfigpool.go

```diff
@@ -52,6 +52,11 @@ func (wt Waiter) ForMachineConfigPoolCondition(ctx context.Context, mcp *machine
 		if err != nil {
 			return false, err
 		}
+		klog.Infof("mcp: updated RV=%q reference RV=%q", updatedMcp.ResourceVersion, mcp.ResourceVersion)
```

Comment:
while only reading the logs (without the code context) it's hard to understand what `RV` is

Reply:
good point, will expand